target/delete.ps1

```diff
@@ -52,11 +56,14 @@ try {
 
     Write-Information 'Verifying if a {connectorName} account exists'
     $correlatedAccount = 'userInfo'
+    # $correlatedAccount = (Invoke-RestMethod @splatGetUserParams) | Select-Object -First 1
 
-    if ($null -ne $correlatedAccount) {
-        $action = 'DeleteAccount'
-    } else {
+    if ($correlatedAccount.Count -eq 0) {
         $action = 'NotFound'
+    } elseif ($correlatedAccount.Count -eq 1) {
+        $action = 'DeleteAccount'
+    } elseif ($correlatedAccount.Count -gt 1) {
+        throw "Multiple accounts found for person where $correlationField is: [$correlationValue]"
     }
```

Comment:
Ik denk dat we deze _in principe_ alleen nodig hebben in de __create__ lifecycle action. In de overige scripts hebben we de accountReference waar we op kunnen zoeken en in de meeste gevallen zal dat een `id` zijn. Overigens mag hij van mij ook blijven staan. We kunnen eventueel ook nog even kijken naar een aantal connectoren en op basis daarvan een keuze maken. Dan kunnen we 'm ook beargumenteren. (Y)